search-app/resources/templates/structure.html

```diff
@@ -8,10 +8,11 @@
 
     {% block head-pre-css %}
     {% endblock %}
-    <!-- Bootstrap core CSS -->
-    <link rel="stylesheet" href="{{ base-url }}site/bootstrap.min.css">
+    <!-- EUI -->
+    <link href="https://cdn.earthdata.nasa.gov/eui/1.1.3/stylesheets/application.css" rel="stylesheet" />
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700,300' rel='stylesheet' type='text/css'>
```

Comment:
We should probably consistently use single or double-quotes throughout. I vote double-quotes.